synapse/push/httppusher.py

```diff
@@ -56,7 +57,9 @@
 )
 
 
-def tweaks_for_actions(actions: List[Union[str, Dict]]) -> Dict[str, Any]:
+def tweaks_for_actions(
+    actions: List[Union[str, Dict]]
+) -> Mapping[str, SimpleJsonValue]:
```

Comment:
I don't see anythign stopping this from being any type of value? Why using `SimpleJsonValue` OK here?

Reply:
Let's use the other thread.

Reply:
OK, so this needs to be broader since we're not limiting it. I think either leaving this as is or using `JsonMapping` is probably OK.